docs/developer/tutorials/snmp/profile-format.md

```diff
@@ -232,7 +232,50 @@ metrics:
         tag: interface
 ```
 
-- Using an "index", i.e. one of the values in the `INDEX` field of the table MIB definition:
+##### Using a column from a different table with different indexes
+
+```yaml
+metrics:
+  - MIB: CPI-UNITY-MIB
+    table:
+      OID: 1.3.6.1.4.1.30932.1.10.1.3.110
+      name: cpiPduBranchTable
+    symbols:
+      - OID: 1.3.6.1.4.1.30932.1.10.1.3.110.1.3
+        name: cpiPduBranchCurrent
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.30932.1.10.1.2.10.1.3
+          name: cpiPduName
+        table: cpiPduTable
+        index_transform:
+          - start: 1
+            end: 7
+        tag: pdu_name
+```
+
+In case the external table have a different indexes we can use `index_transform` to select a subset of the full index we want to use. `index_transform` is a list of `start`/`end` ranges to extract from the current table index to match the external table index. `start` and `end` are inclusive.
```

Comment:
```suggestion
In case the external table has different indexes, we can use `index_transform` to select a subset of the full index we want to use. `index_transform` is a list of `start`/`end` ranges to extract from the current table index to match the external table index. `start` and `end` are inclusive.
```